SPECS/libcgroup/libcgroup.spec

```diff
@@ -1,43 +1,39 @@
 Summary:        Library to control and monitor control groups
 Name:           libcgroup
-Version:        0.41
-Release:        23%{?dist}
+Version:        2.0.1
+Release:        1%{?dist}
 License:        LGPLv2+
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-URL:            https://libcg.sourceforge.net/
-Source0:        https://downloads.sourceforge.net/libcg/%{name}-%{version}.tar.bz2
+URL:            https://github.com/libcgroup/libcgroup
+
+# libcgroup git repo contains submodules that must be part of source tarball
+# 1) clone git repo                           => 'git clone https://github.com/libcgroup/libcgroup.git'
+# 2) checkout tag corresponding to version    => 'git checkout v2.0.1'
+# 3) get submodule                            => 'git submodule init' then 'git submodule update'
+# 4) create source tarball                    => 'tar -czf libcgroup-2.0.1.tar.gz libcgroup'
```

Comment:
Let's make building the tarball reproducible by adding a few more arguments (example from `coredn-1.8.0.spec):

```suggestion
# 4) create source tarball                    => 'tar --sort=name \
#                                                                      --mtime="2021-04-26 00:00Z" \
#                                                                      --owner=0 --group=0 --numeric-owner \
#                                                                      --pax-option=exthdr.name=%d/PaxHeaders/%f,delete=atime,delete=ctime \#                                                                      -czf %%{name}-%%{version}.tar.gz %%{name}'
```

Reply:
For some reason the spacing got messed up between the edit and preview.:/